src/js/pensions/config/form.js

```diff
@@ -176,9 +176,8 @@ function createSpouseLabelSelector(nameTemplate) {
 
 const formConfig = {
   urlPrefix: '/',
-  submitUrl: '/v0/pension_claims',
+  submit,
   trackingPrefix: 'pensions-527EZ-',
-  transformForSubmit: transform,
```

Comment:
calling this from inside of the helper functions- it doesn't need to be passed to the form anymore